src/index.js

```diff
@@ -1 +1,83 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import { createGame, addScore, getScores } from './modules/api.js';
+import { storeScores } from './modules/localstorage.js';
+
+const submitForm = document.querySelector('.add_score');
+const refreshButton = document.querySelector('.refresh button');
+const scoresContainer = document.querySelector('.scores-container');
+const errorMsg = document.querySelector('.error-msg');
+
+let chessGameId;
+let submittedScores = [];
+
+function refreshScores(scoresArray) {
+  scoresContainer.innerHTML = '';
+
+  scoresArray.sort((a, b) => b.score - a.score);
+
+  scoresArray.forEach((scoreObj) => {
+    const scoreElement = document.createElement('p');
+    scoreElement.textContent = `${scoreObj.user}: ${scoreObj.score}`;
+    scoresContainer.appendChild(scoreElement);
+  });
+}
+
+submitForm.addEventListener('submit', async (event) => {
+  event.preventDefault();
+  const nameInput = document.querySelector("input[placeholder='Your name']");
+  const scoreInput = document.querySelector("input[placeholder='Your score']");
+
+  const name = nameInput.value.trim();
+  const score = parseInt(scoreInput.value, 10);
+
+  if (!name || !score) {
+    errorMsg.style.display = 'block';
+    return;
+  }
+
+  try {
+    await addScore(chessGameId, name, score);
+
+    nameInput.value = '';
+    scoreInput.value = '';
+
+    errorMsg.style.display = 'none';
+    scoresContainer.style.display = 'block';
+
+    const newScore = { user: name, score };
+    submittedScores.push(newScore);
+
+    storeScores(submittedScores);
+    refreshScores(submittedScores);
+  } catch (error) {
+    // Handle the error
+  }
+});
+
+refreshButton.addEventListener('click', async () => {
+  errorMsg.style.display = 'none';
+  scoresContainer.style.display = 'block';
+
+  try {
```

Comment:
- [ ]  I encountered the same issue that was flagged up by the previous reviewer. When I attempted to test the functionality of the refresh button, I noticed that data was being posted directly to the UI upon clicking the submit button. This resulted in an error, as the data should be sent to the API instead.  Check in the video below :point_down:  and fix that 

https://github.com/mryeminthu/Leaderboard/assets/88894525/939f4c21-669c-4c5e-bb7f-068db03016ed